### ISSUE TITLE
ideapad 710s ahci/raid

### ISSUE THREAD

**Issue by gulci-poz** [trepmag/710s-13ikb#2]
Hi,
Is it possible to choose RAID mode and to boot Windows after BIOS upgrade? On Lenovo forum Kunter wrote that after BIOS upgrade you have to switch to AHCI. Lenovo states what follows:
"This BIOS creates an AHCI SATA Controller Mode selection to allow for the installation of the Linux operating system.
This BIOS is NOT intended for use with machines running a Windows operation system.
No official support is provided for this BIOS. The user assumes the risk of any performance degradation associated with the use of this BIOS."
I was wondering if there's any possibility of rolling back or upgrading to the main branch BIOS, as the upgrading tool is available as .exe file or if there'e possibility of going back to Windows, for example for reselling purposes.
Greetings,
Sebastian

**Comment by lucazade** [comment]
Yes, there is an option in the bios to switch from RAID <> AHCI after the linux bios upgrade and yes you can dual boot windows and linux with grub.

You can avoid to switch the two modes everytime to boot the two OS. You can "convert" Windows 10 in AHCI mode (without reistalling it), just follow this guide:
http://triplescomputers.com/blog/uncategorized/solution-switch-windows-10-from-raidide-to-ahci-operation/

**Comment by gulci-poz** [comment]
Great! Thanks.
